std_msgs/Header header

#state of IMU:
bool imu_active
bool imu_setteled
bool imu_terminate_flag

sensor_msgs/Imu imu

float32 yaw
float32 roll
float32 pitch
